lib/semantics/tools.cc

```diff
@@ -264,6 +265,19 @@ bool ExprHasTypeCategory(const evaluate::GenericExprWrapper &expr,
   return dynamicType.has_value() && dynamicType->category == type;
 }
 
+bool ExprHasTypeKind(const evaluate::GenericExprWrapper &expr, int kind) {
```

Comment:
Is this used?

Reply:
Nope. Removed. Thanks for spotting this.